packages/bpk-component-nudger/src/BpkNudger-test.js

```diff
@@ -57,6 +58,27 @@ describe('BpkNudger', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('should render as a configurable nudger correctly', () => {
+    const tree = renderer
+      .create(
+        <BpkConfigurableNudger
+          id="nudger"
+          min={1}
+          max={9}
```

Comment:
Could we make this test do something other than numbers? Just because it's not really that different from the other tests otherwise. String should be good